context/app/static/js/schema.org.js

```diff
@@ -0,0 +1,43 @@
+function getDatasetLD(entity) {
+  // Based on https://developers.google.com/search/docs/data-types/dataset#guidelines
+  const donor = entity.donor.mapped_metadata;
+  const shortDonorString = `${donor.sex || '(Unknown sex)'}, ${donor.age_value || '(Unknown age)'} ${
+    donor.age_unit || '(Unknown age unit)'
+  } old`;
+  const heightString = `${donor.height_value || ''}${donor.height_unit || ''}`;
+  const weightString = `${donor.weight_value || ''}${donor.weight_unit || ''}`;
+  const longDonorString = `${heightString}, ${weightString}, ${donor.race || ''} ${shortDonorString}`;
```

Comment:
Could this have a lot of extra spaces if heightString, weightString or race are undefined?

Reply:
yes, it could, but my sense is that these will be provided for every donor, so I don't want to make the code more complicated than it needs to be. I'm happy with this... but if you think it could be less fragile, happy to do so.